src/Components/SearchForm/SearchForm.js

```diff
@@ -81,6 +81,7 @@ const SearchForm = (props) => {
           <option value="interlocks+sisterlocks">interlocks</option>
           <option value="microlocs">microlocs</option>
           <option value="faux locs">faux locs</option>
+          <option value="loc+extensions">permanent loc extensions</option>
```

Comment:
I opted for "locs+extensions" because I saw in my search that permanent loc extensions were showing up. If I see later that they tend to include faux locs as well, I will update the search term to include the word "permanent" as well.